composer.json

```diff
@@ -18,20 +18,20 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^5.6 || ^7.0",
-        "laminas/laminas-config": "^2.0",
-        "laminas/laminas-crypt": "^2.0",
-        "laminas/laminas-http": "^2.0",
-        "laminas/laminas-i18n": "^2.0",
-        "laminas/laminas-loader": "^2.0",
-        "laminas/laminas-math": "^2.0",
-        "laminas/laminas-stdlib": "^2.7.8",
-        "laminas/laminas-uri": "^2.0",
+        "php": "^7.3 || ~8.0",
+        "laminas/laminas-config": "^2.6 || ^3.0",
+        "laminas/laminas-crypt": "^2.6 || ^3.0",
+        "laminas/laminas-http": "^2.14",
+        "laminas/laminas-i18n": "^2.11",
+        "laminas/laminas-loader": "^2.7",
+        "laminas/laminas-math": "^2.7 || ^3.3",
+        "laminas/laminas-stdlib": "^2.7.8 || ^3.2",
+        "laminas/laminas-uri": "^2.8",
```

Comment:
Is this intended that we keep old version compatible?

Reply:
That's a good point - I'll drop those.